src/shared-components/optionButton/index.tsx

```diff
@@ -43,35 +98,49 @@ export const OptionButton = ({
     // eslint-disable-next-line react/jsx-props-no-spreading
     {...rest}
   >
-    <Style.FlexContainer>
-      {/**
-       * We sometimes use && conditionals such that we are passing in `false` as a value
-       */}
-      {isDefined(icon) && icon !== false ? (
-        <Style.IconWrapper
-          selected={selected}
-          optionType={optionType}
-          buttonType={buttonType}
-        >
-          {selected ? <CheckmarkIcon /> : icon}
-        </Style.IconWrapper>
-      ) : (
-        <Style.CheckmarkWrapper
-          selected={selected}
-          optionType={optionType}
-          buttonType={buttonType}
-        >
-          <CheckmarkIcon />
-        </Style.CheckmarkWrapper>
-      )}
-      <Style.TextContainer>
-        <Style.Text>{text}</Style.Text>
-        {isDefined(subtext) && <Style.SubText>{subtext}</Style.SubText>}
-      </Style.TextContainer>
-    </Style.FlexContainer>
+    <OptionButtonContent
+      buttonType={buttonType}
+      icon={icon}
+      optionType={optionType}
+      selected={selected}
+      subtext={subtext}
+      text={text}
+    />
   </Style.ClickableContainer>
 );
 
+/**
+ * A presentational component to match the display of an OptionButton with an icon
+ */
+export const OptionButtonContainer = ({
+  icon,
+  optionType,
+  subtext,
+  text,
+  ...rest
+}: OptionButtonContainerProps) => (
+  <Style.DisplayContainer
+    containerType="none"
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    {...rest}
+  >
+    <OptionButtonContent
+      // The buttonType does not matter for this component
+      buttonType="primary"
+      icon={icon}
+      optionType={optionType}
+      selected={false}
+      subtext={subtext}
+      text={text}
+    />
+  </Style.DisplayContainer>
+);
+
+/**
+ * Similar OptionButton styling without click elements
+ */
+OptionButton.Container = OptionButtonContainer;
```

Comment:
I'm open to renaming this, since `Container` is typically used in our codebase (like in the `Accordion`) to wrap the component and apply uniform styling. I wouldn't even be against something extremely clear, like `OptionButton.NoClickFunctionality`. I would prefer it be a separate component like that than a prop.

Reply:
Good call - I would vote for the explicit name over `Container`. Maybe `OptionButton.NotClickable`?  

I'm not sure if there's a name that matches this UI concept - semantic-ui-react has a `Segment` [component](https://react.semantic-ui.com/elements/segment/) that looks kind of similar, but I think that's a different concept.

Reply:
I think there is a way to extract a more obviously non-functional element out of this by expanding our main `Container` component. The problem is the usage (and presentation) matches so obviously to the `OptionButton` that it seems obfuscating to treat it as otherwise.

Given this implementation is on-the-go I'm comfortable waiting until we see a clearer pattern emerge from usage before placing the logic outside of the `OptionButton`.